topostats/run_topostats.py

```diff
@@ -255,7 +255,7 @@ def process_scan(
     if grains.region_properties is not None:
         # Grain Statistics :
         try:
-            LOGGER.info(f"[{filtered_image.filename}] : Grain Statistics")
+            LOGGER.info(f"[{filtered_image.filename}] : *** Grain Statistics ***")
```

Comment:
The "***...***" is not consistent across processing, grain finding and grain statistics. 

Is this by design?